samples/nrf9160/modem_shell/Kconfig

```diff
@@ -0,0 +1,79 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "Kconfig.zephyr"
+
+menu "Nordic Modem Shell config"
+
+config MOSH_IPERF3
+	bool  "Enable iperf3 support"
+	default y
+	select NRF_IPERF3
+
+config MOSH_SOCK
+	bool  "Enable socket tool support"
+	default y
+
+config MOSH_PING
+	bool  "Enable ICMP ping support"
+	default y
+
+config MOSH_LINK
+	bool  "Enable link control support"
+	depends on LTE_LINK_CONTROL
+	depends on SETTINGS
+	default y
+
+config MOSH_CURL
+	bool  "Enable CURL"
+	default y
+	select NRF_CURL
+
+config MOSH_GNSS
+	bool  "Enable GNSS support"
+	default y
+
+config MOSH_PPP
+	bool  "Enable PPP support"
+
+config MOSH_SMS
+	bool  "Enable SMS support"
+	default y
+
+config MOSH_FOTA
+	bool "Enable FOTA support"
+	default y
+
+if MOSH_LINK
+menu "MoSH link command selections"
+
+config MOSH_LINK_SETT_NORMAL_MODE_AT_CMD_STR_LEN
+	int "configure string length for custom AT cmds stored in settings and used when going to normal mode"
```

Comment:
Could you please shorten this prompt.

Use the help text for detailed explanation.

Reply:
Fixed